File: C_Programming/Competitive/Question31.c
// write a program which accept number from user and display below pattern.

// input :  5
// output : *   *   *   *   *   #   #   #   #   #

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1 ; iCnt <= iNo ;iCnt++)
    {
        printf("*   ");
    }

    for(iCnt = 1 ; iCnt <= iNo ;iCnt++)
    {
        printf("#   ");
    }

}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

// Time complexity : O(N)

File: Python_Programming/Competitive/Question31.py
'''
        write a lambda function which accepts one number and return True if number if odd otherwise False
'''

ChkOdd = lambda  No: True if No % 2 == 1 else False

def main():
    value = int(input("Enter number : "))

    Ret = ChkOdd(value)

    if Ret == True:
        print("True")
    else:
        print("False")

if __name__ == "__main__":
    main()